### DISCUSSION TITLE
Milvus standalone server with GPU support

### DISCUSSION THREAD

**Discussion by kraza8** [milvus-io/milvus#30594] [Q&A and General discussion]
As per this link: https://milvus.io/docs/install_standalone-helm-gpu.md 
"Milvus now can use GPU devices to build indexes and perform ANN searches ..."

Can the GPU acceleration be used for Milvus **standalone server**? (no Pulsar cluster and no Kubernetes cluster).

If this is possible, what vector database operations can be offloaded to the GPU? e.g. vector embedding, vector normalization, indexing etc.?

Thanks

**Comment by yhmo** [comment]
Use docker to deploy a GPU standalone:
https://milvus.io/docs/install_standalone-docker-compose-gpu.md


Milvus is mainly for massive vectors similarity search, vectors are indexed to achieve high search performance. It has no ability to create vectors or normalize vectors.
